kubernetes/charts/weblogic-operator/templates/_operator-dep.tpl

```diff
@@ -36,6 +36,8 @@ spec:
           value: "false"
         - name: "JAVA_LOGGING_LEVEL"
           value: {{ .javaLoggingLevel | quote }}
+        - name: ISTIO_ENABLED
+          value: {{ .istioEnabled | quote }}
```

Comment:
Why is this needed?  Can the value instead be derived from the domain resource as needed?

Reply:
This is required in the create-domain step when the domain resource has not been generated.

Reply:
Will this have a negative impact on domains that don't set the experimental istio flags?

Reply:
It is harmless. The property istioEnabled is defined in "kubernetes/charts/weblogic-operator/values.yaml" and its value is false by default.